snippets/javascript/react-es7.json

```diff
@@ -1035,11 +1059,17 @@
         "prefix": "rnf",
         "body": [
             "import { View, Text } from 'react-native'",
+<<<<<<< HEAD
+            "import React from 'react'",
+            "",
+            "export default function ${TM_FILENAME_BASE}() {",
+=======
```

Comment:
Could you double check the conflicts @ydkulks, I'm still seeing a few merge conflict markers in the diff.